contracts/integrations/test/coordinator/coordinator_test.ts

```diff
@@ -53,10 +64,10 @@ blockchainTests.resets('Coordinator integration tests', env => {
             orderConfig: {
                 senderAddress: coordinator.address,
                 feeRecipientAddress: feeRecipient.address,
-                makerAssetData: assetDataUtils.encodeERC20AssetData(makerToken.address),
-                takerAssetData: assetDataUtils.encodeERC20AssetData(takerToken.address),
-                makerFeeAssetData: assetDataUtils.encodeERC20AssetData(makerFeeToken.address),
-                takerFeeAssetData: assetDataUtils.encodeERC20AssetData(takerFeeToken.address),
+                makerAssetData: await devUtils.encodeERC20AssetData.callAsync(makerToken.address),
```

Comment:
Do you have any sense of how much this will increase build times? I understand the desire to remove `assetDataUtils` from a maintenance perspective, but it does seem like the ability to encode asset data without making an RPC call is nice from a performance perspective. Thoughts?

Reply:
Is it not the case that these would be executed in-process rather than RPC due to us using ethereumjs-vm?  Or did we not yet merge that in?  (I keep getting confused on where we are with that, given the mixed resistance to it.)

In any case, I took some measurements:

```bash
0x-monorepo[development]$ time yarn test:contracts
real	19m28.792s
user	19m42.732s
sys	1m42.184s
```

```bash
0x-monorepo[2137/trim-order-utils]$ time yarn test:contracts
real	24m18.599s
user	24m46.280s
sys	1m54.212s
```

(Note that I rebased this PR's branch on `development` before measuring, just so it would be as apples-to-apples as possible.)

Reply:
Nice, thanks for doing that @feuGeneA . 

Yeah, all pure functions in typescript will be executed in process with ethereumjs-vm. Looks like it is about 25% slower than doing it in pure ts.

Reply:
@feuGeneA Thanks for the benchmarking, and the reminder that we are using the ethereumjs-vm for pure functions.

Reply:
Hmm, that is a fairly significant slowdown. I may bring some of the code in `assetDataUtils` back into `contracts-test-utils` in a later PR.